### ISSUE TITLE
Choices.js doesn't select value from binded model

### ISSUE THREAD

**Issue by jahazzz** [protonemedia/laravel-splade#163]
Hi (again),

I have a form like this :
```
<x-splade-form action="{{route('sales.taxes.tax-zones.tax-rates.update', [$taxZone, $taxRate])}}" :default="$taxZone" method="put">

    <x-splade-select name="tax_class_id" remote-url="{{ route('api.sales.taxes.tax-classes.index') }}" choice relation />

</x-splade-form>
```

$taxZone->tax_class_id = 1 but there is no selected value in the select : 
![Capture d’écran 2022-12-22 à 14 46 36](https://user-images.githubusercontent.com/2676572/209165281-fff8bf5e-3d6c-4d36-b801-6ca099ed188c.png)

If I remove "choice" : 
![Capture d’écran 2022-12-22 à 14 46 12](https://user-images.githubusercontent.com/2676572/209165689-185b58ec-19a2-40f0-8a8c-58a1ceaf3024.png)

I tried without remote-url and the result still the same.

And thanks for the quick response to my previous issue [#154](https://github.com/protonemedia/laravel-splade/issues/154), but your update doesn't work, see my comment in that issue.

**Comment by stevegitau** [comment]
I have the same issue with choices but it works fine when choice option is removed. The problem is some forms are too big/long and hence would be a better choice if the bug is fixed.

**Comment by jahazzz** [comment]
I found that if you cast the model id to string, Choices selects properly the value but it's a boring workaround.

**Comment by pascalbaljet** [comment]
Yes, this is a limitation of Choices.js. In the backend, the values are already [casted to strings](https://github.com/protonemedia/laravel-splade/blob/85685f54828338173233bf736159c19b2243ad79/src/Components/Form/Select.php#L134-L156), but not with remote options. I'll add it to remote options as well.

**Comment by pascalbaljet** [comment]
Fixed in v1.1.4. Make sure to [update both packages](https://splade.dev/docs/upgrading) (JS + PHP).
